frontend/components/commons/SideBar.vue

```diff
@@ -5,101 +5,81 @@
-            @click="closeModal()"
-          >
-            Cancel
-          </ReButton>
-          <ReButton
-            class="button-secondary--small"
-            @click="onExportAnnotations()"
-          >
-            Confirm
+          <ReButton v-if="annotationIsEnabled" class="button-clear button-action global-actions__export" @click="onOpenExportModal()">
+            <svgicon name="export" width="14" height="14" color="#0508D9" />Create snapshot
           </ReButton>
+          <ReModal v-if="annotationIsEnabled" :modal-custom="true" :prevent-body-scroll="true" modal-class="modal-primary" :modal-visible="openExportModal" modal-position="modal-center" @close-modal="closeModal()">
+            <p class="modal__title">Confirm snapshot creation</p>
+            <p class="modal__text">
+              You are about to export {{ annotationsSum }} annotations. You will find the file on the server once the action is completed.
+            </p>
+            <div class="modal-buttons">
+              <ReButton class="button-tertiary--small button-tertiary--outline" @click="closeModal()">
+                Cancel
+              </ReButton>
+              <ReButton class="button-secondary--small" @click="onExportAnnotations()">
+                Confirm
+              </ReButton>
+            </div>
+          </ReModal>
         </div>
-      </ReModal>
-    </div>
-    <!-- <div v-else-if="dataset.results.aggregations.words" class="sidebar__wrapper">
-      <div class="sidebar__content">
-        <p><svgicon name="metrics" width="24" height="24" color="#4C4EA3" /> Keywords</p>
-        <svgicon class="sidebar__close-button" @click="toggleVisibleMetrics" name="cross" width="16" height="16" color="#4C4EA3" />
-        <div class="scroll">
-          <div
-            v-for="(counter, label) in dataset.results.aggregations.words"
-            :key="label"
-          >
-            <div v-if="counter > 0" class="info">
-              <label>{{ label }}</label>
-              <span class="records-number">{{ counter }}</span>
+        <div v-else>
+          <div class="sidebar__tabs">
+            <a href="#" :class="activeTab === 'mentions' ? 'active' : ''" @click.prevent="filteredMentionsBy('mentions')">Annotated as</a>
+            <a href="#" :class="activeTab === 'predicted_mentions' ? 'active' : ''" @click.prevent="filteredMentionsBy('predicted_mentions')">Predicted as</a>
+          </div>
+          <div class="scroll">
+            <div v-for="(prop, key) in filteredMentions" :key="key" :class="expandedMentionsGroup === key ? 'expanded' : ''">
+              <span :class="[`color_${entities.filter(e => e.text === key)[0].colorId}`, 'entity']">{{ key }}</span>
+              <SidebarCollapsableMentions :limit="expandedMentionsGroup && expandedMentionsGroup !== key ? 0 : currentMentionsLength" :entities="entities" :k="key" :object="filteredMentions" @limit="onShowMore(key)" />
```

Comment:
If the `SideBar` is a common component, we should hide specific task properties references (like mentions). Can we move the mentions logic into an specific sidebar component for token classification?

Reply:
@frascuchon I have separated global sidebar and two specific task based components